src/app/components/Header/__snapshots__/index.test.jsx.snap

```diff
@@ -16,7 +28,39 @@ exports[`Header should render correctly 1`] = `
   <a
     href="https://www.bbc.co.uk/news"
   >
-    BBC News
+    <svg
+      aria-hidden="true"
+      focusable="false"
+      style={
+        Object {
+          "height": "40px",
```

Comment:
By adding the style in the `Header/index.jsx` file via the `svgStyle` object, in the output here you get the object rather than key value pairs.  It will likely be better to add the attributes directly with the other SVG attributes like viewBox.

Reply:
I thought I might get that in a review :) - will update now.

Reply:
I've reverted this change so this ticket isn't adding style and is focusing instead on the semantics the inline SVG should accomplish. 

The styling will be addressed in https://github.com/bbc/simorgh/issues/346

Therefore my comment earlier about the SVG needing to be styled can be ignored.